build.gradle

```diff
@@ -1,8 +1,270 @@
-// Uses Gradle to build RPMs since that's what we use for the other plugins. When all you have is a hammer...
+buildscript {
+    ext {
+        es_version = System.getProperty("es.version", "7.10.2")
+    }
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            url = uri('https://repo.maven.apache.org/maven2/')
+        }
+    }
+
+    dependencies {
+        classpath "org.elasticsearch.gradle:build-tools:${es_version}"
+    }
+}
+
 plugins {
-   id "nebula.ospackage" version "5.3.0"
+    id 'java'
+    id 'nebula.ospackage' version '8.5.6'
+    id "com.gorylenko.gradle-git-properties" version "2.2.4"
+}
+
+apply plugin: 'elasticsearch.esplugin'
+esplugin {
+    name 'opendistro_security'
+    description 'Open Distro Security for Elasticsearch'
+    classname 'com.amazon.opendistroforelasticsearch.security.OpenDistroSecurityPlugin'
 }
 
+group = 'com.amazon.opendistroforelasticsearch'
+version = '1.13.1.0'
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+bundlePlugin {
+    archiveName("opendistro-security-${version}.zip")
+
+    from("securityconfig") {
+        into "securityconfig"
+    }
+    from("tools") {
+        into "tools"
+    }
+    from("plugin-security.policy") {
+        into "."
+    }
+}
+
+group = 'com.amazon.opendistroforelasticsearch'
+version = '1.13.1.0'
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+apply plugin: 'distribution'
+distributions {
+    custom {
+        distributionBaseName = "opendistro-security-${version}-securityadmin-standalone"
+        contents {
+            into('./tools/') {
+                from 'tools'
+            }
+            into('./deps/securityconfig') {
+                from 'securityconfig'
+            }
+        }
+    }
+    customDistZip.setArchiveName("opendistro-security-${version}-securityadmin-standalone.zip")
+    customDistTar.setArchiveName("opendistro-security-${version}-securityadmin-standalone.tar.gz")
+}
+
+customDistTar{
+    compression = Compression.GZIP
+    archiveExtension = "tar.gz"
+}
+
+ext {
+    projectSubstitutions = [:]
+    licenseFile = rootProject.file('LICENSE.txt')
+    noticeFile = rootProject.file('NOTICE.txt')
+}
+
+configurations.all {
+    if (it.state != Configuration.State.UNRESOLVED) return
+    resolutionStrategy {
+        force 'commons-codec:commons-codec:1.14'
```

Comment:
Do all the dependencies need force?

Reply:
Yes, all these need force because otherwise some dependency conflicts come out.